compiler/semantic/op.go

```diff
@@ -20,22 +20,29 @@ import (
 	"golang.org/x/exp/slices"
 )
 
-func semFrom(ctx context.Context, scope *Scope, from *ast.From, source *data.Source, head *lakeparse.Commitish) (*dag.From, error) {
-	var trunks []dag.Trunk
-	for _, in := range from.Trunks {
-		converted, err := semTrunk(ctx, scope, in, source, head)
-		if err != nil {
-			return nil, err
+func semFrom(ctx context.Context, scope *Scope, from *ast.From, source *data.Source, head *lakeparse.Commitish) (dag.Op, error) {
+	switch len(from.Trunks) {
+	case 0:
+		return nil, errors.New("from operator has no paths")
```

Comment:
This is an internal error, right?
```suggestion
		return nil, errors.New("internal error: from operator has no paths")
```